src/language/providers/attributeCompletion.ts

```diff
@@ -28,38 +28,25 @@ import {
 } from './utils'
 
 import { attributeCompletion } from './intellisense/attributeItems'
+import { getCommonItems, createCompletionItem } from './utils'
 
 function getCompletionItems(
   itemsToUse: string[],
   preVal: string = '',
   additionalItems: string = ''
 ) {
-  let compItems: vscode.CompletionItem[] = []
-  let noPreVals: string[] = [
-    'dfdl:choiceBranchKey=',
-    'dfdl:representation',
-    'dfdl:choiceDispatchKey=',
-    'dfdl:simpleType',
-    'xs:restriction',
-  ]
+  let compItems: vscode.CompletionItem[] = getCommonItems(
+    itemsToUse,
+    preVal,
+    additionalItems
+  )
 
+  compItems.forEach((i) =>
+    vscode.window.showInformationMessage(i.label.toString())
+  )
```

Comment:
Is this correct? This feels like some sort of debug output?

Reply:
Yeah that is what it is I forgot to delete will remove that